colors.py

```diff
@@ -0,0 +1,31 @@
+import random
+class Colors:
+    def __init__(self):
+        self.SECONDARY = (0,0,0)
+        self.PRIMARY = (255,255,255)
+        self.TERTIARY = (128, 128, 128)
+
+    _colors = (
+        (250, 78, 233),     # purple pizzazz
+        (61, 226, 221),    # turquoise
+        (255, 132, 74),       # mango tango
+        (141, 255, 62),      # spring frost
+        (221, 29, 96),      # ruby
+        (157, 57, 238),      # purple x 11
+        (31, 255,186),      # sea green crayola
+        (147,255,255),       # electric blue
+        (255,175,25)        # honey yellow
+    )

```

Comment:
This is great, but I think we can improve the naming using Enums instead of putting it in comments.
You can simply import `Enum` and maybe you can make `Colors` extend it.

So something like
```
class Colors(Enum):
     RUBY = (221, 29, 96)
```
What do you think?

Reply:
The comments are just for the curious, the names of the colors are not needed.

Reply:
I think it would be a good refactoring for the future, but looks good to me other than that.